src/components/Autosuggest/Autosuggest.js

```diff
@@ -21,17 +21,21 @@ const { block, elem } = bem({
 });
 
 class Autosuggest extends React.Component {
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
         this.inputRef = React.createRef();
         this.rootRef = React.createRef();
         this.listRef = React.createRef();
         this.state = {
             inputValue: '',
             inputValueRecall: '',
-            focused: false,
+            focused: props.isFocused,
         };
 
+        if (props.isFocused) {
+            this.focus();
+        }
```

Comment:
I suggest to move this code in **componentDidMount** and then is possible to get rid of **setTimeout** in focus method